patches/net/minecraft/world/item/Item.java.patch

```diff
@@ -148,31 +148,31 @@
      }
  
 -    public static class Properties {
-+    // NEO START
++    // Neo: Client rendering for Items
 +    private Object renderProperties;
 +
-+    /*
-+        DO NOT CALL, IT WILL DISAPPEAR IN THE FUTURE
-+        Call RenderProperties.get instead
-+    */
++    /**
++     * Neo: DO NOT CALL, IT WILL DISAPPEAR IN THE FUTURE
```

Comment:
Should we TODO this for removal?

Reply:
Added todo on all of these so we can track them in future easier